com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -575,11 +575,6 @@ public BufferedImage generateGlyphData(boolean preview, final FontResourceResolv
             cell_max_descent = Math.max(cell_max_descent, descent);
         }
 
-        // Make sure it fits future glyphs (provided at runtime)
-        cell_max_ascent  = (int)Math.ceil(Math.max(cell_max_ascent, glyphBankBuilder.getMaxAscent()));
-        cell_max_descent = (int)Math.ceil(Math.max(cell_max_descent, glyphBankBuilder.getMaxDescent()));
-        cell_width = (int)Math.ceil(Math.max(cell_width, glyphBankBuilder.getMaxWidth()));
```

Comment:
Getting max size from a font doesn't work well. The result is simply too large, wasting a lot of space in the glyph cache texture.

Removing this block of code reverts the functionality back to the way it was before my change.